stable/sumologic-fluentd/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: sumologic-fluentd
-version: 0.8.2
+version: 0.8.3
```

Comment:
since the PR adds a new config parameter to the chart, you should bump the minor version here